src/ansi-c/c_typecheck_type.cpp

```diff
@@ -93,6 +93,9 @@ void c_typecheck_baset::typecheck_type(typet &type)
     typecheck_symbol_type(to_symbol_type(type));
   else if(type.id() == ID_typedef_type)
     typecheck_typedef_type(type);
+  else if(type.id() == ID_union_tag)
+  {
+  }
```

Comment:
This deserves a comment that it is intentionally left blank (explaining where else the work is done, or that no work is necessary).

Reply:
Done